src/windows/ReactXP.ts

```diff
@@ -66,6 +66,7 @@ AccessibilityUtil.setAccessibilityPlatformUtil(AccessibilityPlatformUtil);
 // -- STRANGE THINGS GOING ON HERE --
 // See web/ReactXP.tsx for more details.
 
+// TODO change
```

Comment:
What is this TODO for?

Reply:
To... remove itself!

---
In reply to: [159358539](https://github.com/Microsoft/reactxp/pull/404#discussion_r159358539) [](ancestors = 159358539)